frontend-react/src/hooks/UseLookupTable.test.ts

```diff
@@ -0,0 +1,63 @@
+import { renderHook } from "@testing-library/react-hooks";
+
+import { lookupTableServer } from "../__mocks__/LookupTableMockServer";
+import { LookupTables, ValueSet } from "../network/api/LookupTableApi";
+
+import useLookupTable, {
+    generateUseLookupTable,
+    GetLatestData,
+    GetLatestVersion,
+    getSenderAutomationData,
+} from "./UseLookupTable";
+
+jest.mock("@okta/okta-react", () => ({
+    useOktaAuth: () => {
+        const authState = {
```

Comment:
Check with @doug-s-nava , or take a peek at some of the other Okta mocks; we have a new way to mock this out and shouldn't continue this pattern.

Reply:
@kevinhaube as it turns out, I didn't even need this mock! just a copy/paste artifact.